nri-config-generator/integration-tests/generator_test.go

```diff
@@ -102,24 +102,25 @@ func TestMain(m *testing.M) {
 		log.Fatal(err)
 	}
 	exitVal := m.Run()
-	clean()
 	os.Exit(exitVal)
 }
 
-/**
+/*
+*
```

Comment:
Just out of curiosity, are we using this kind of comments for any particular reason?

Reply:
nope, format-on-save changed this but i'm going to change it to line comments